ci/workflows/integration-tests.yml

```diff
@@ -79,6 +80,10 @@ steps:
         #    testcase: "iceberg-sink"
         #    format: "protobuf"
         #  skip: true
+        - with:
+            testcase: "iceberg-cdc"
+            format: "protobuf"
+          skip: true
```

Comment:
What's this `skip: true` means? cc @neverchanje

Reply:
In this matrix, it will execute `iceberg-cdc,protobuf`, `iceberg-cdc,json`. We just skip one.